pkg/agent/config/config.go

```diff
@@ -74,6 +76,19 @@ func GetConfig(mode Mode) (*Config, error) {
 	return config, nil
 }
 
+func setLogLevel() {
+	agentLogLevel := os.Getenv("AGENT_LOG_LEVEL")
+	if agentLogLevel != "" {
+		level, err := log.ParseLevel(agentLogLevel)
+		if err != nil {
+			log.Fatalf("Invalid log level: %s", agentLogLevel)
```

Comment:
No need for a fatal error here, log with `Errorf` and defalt to the info level

Reply:
You mean this: `log.Errorf("Invalid log level: %s, defaulting to info level", agentLogLevel)`

Reply:
Yes

Reply:
Done!